Tools/i18n/pygettext.py

```diff
@@ -742,14 +1006,18 @@ class Options:
     # initialize list of strings to exclude
     if options.excludefilename:
         try:
-            with open(options.excludefilename) as fp:
-                options.toexclude = fp.readlines()
+            options.toexclude = get_msgids_from_exclude_file(
```

Comment:
It seems that in `xgettext` you can specify `--exclude-file` multiple times, and all msgids are added.